core/src/main/scala/com/salesforce/op/filters/Summary.scala

```diff
@@ -39,18 +40,38 @@ import com.twitter.algebird.Monoid
  * @param max   maximum value seen for double, maximum number of tokens in one text for text
  * @param sum   sum of values for double, total number of tokens for text
  * @param count number of doubles for double, number of texts for text
+ * @param maxCardinality maximum number of unique tokens to keep track of, for a given text feature
  */
-case class Summary(min: Double, max: Double, sum: Double, count: Double)
+case class Summary(min: Double, max: Double, sum: Double, count: Double,
+                   textLength: Option[Moments] = None,
+                   textCard: Option[TextStats] = None,
+                   maxCardinality: Int = 500)
```

Comment:
Can we put this value in the `RawFeatureFilter` companion object and pull it from there to keep the concrete thresholds are in one place?

Reply:
Noted, will move this to a more sensible place once I confirm that the metrics work correctly.